### DISCUSSION TITLE
Being able to add relationship or relationship.field<string> to searchFields

### DISCUSSION THREAD

**Discussion by AhmedBG** [keystonejs/keystone#8080] [Feature requests]
### How to reproduce the problem

When introducing the `searchFields`, I wanted to use it to be able to search by `relationship`. 

- Using the `relationship` inside the `searchFields` I get this error in the logs:
```
🚨 There was an error loading your Keystone config
Error: The ui.searchFields option on the ClientShop list includes 'clientProject' but that field doesn't have a contains filter that accepts a GraphQL String
    at Object.createAdminMeta (/home/matcha/Desktop/MatchaCMS/node_modules/@keystone-6/core/dist/createAdminMeta-2ef83025.cjs.dev.js:136:15)
    at Object.createSystem (/home/matcha/Desktop/MatchaCMS/node_modules/@keystone-6/core/dist/createSystem-5796fcbd.cjs.dev.js:2229:37)
    at initKeystone (/home/matcha/Desktop/MatchaCMS/node_modules/@keystone-6/core/scripts/cli/dist/keystone-6-core-scripts-cli.cjs.dev.js:666:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

- When trying to add the relationship.field<string>, I get no error and nothing happens either!

### What I expected to happen

I think adding a `relationship` into the `searchFields` should automatically search by the `labelField` of the `relationship`, or being able to specify what field inside the relationship we wanna search by using this format (`relationship.field<string>`)

**Comment by dcousens** [comment]
> Error: The ui.searchFields option on the ClientShop list includes 'clientProject' but that field doesn't have a contains filter that accepts a GraphQL String

Does the field type used by `clientProject` have a `contains` filter that accepts a GraphQL string?

**Comment by AhmedBG** [comment]
> Does the field type used by `clientProject` have a `contains` filter that accepts a GraphQL string?

No, `clientProject` is a relationship, it has only [this filters](https://keystonejs.com/docs/graphql/filters#relationship). But it would be nice to be able to search by relationship id or label

**Reply by dcousens** [reply]
Interesting idea!  I'm going to change this issue into a feature request for now though, as this isn't a bug

**Reply by namoey** [reply]
Are there any updates here? This feature will be very helpful since searching with the field of relationships is actually a common case.
